addons/voice/org.openhab.voice.gtts/src/main/java/org/openhab/voice/gtts/internal/GoogleTTSService.java

```diff
@@ -0,0 +1,363 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.voice.gtts.internal;
+
+import com.google.cloud.texttospeech.v1beta1.AudioEncoding;
+import org.eclipse.smarthome.config.core.ConfigConstants;
+import org.eclipse.smarthome.config.core.ConfigOptionProvider;
+import org.eclipse.smarthome.config.core.ConfigurableService;
+import org.eclipse.smarthome.config.core.ParameterOption;
+import org.eclipse.smarthome.core.audio.AudioException;
+import org.eclipse.smarthome.core.audio.AudioFormat;
+import org.eclipse.smarthome.core.audio.AudioStream;
+import org.eclipse.smarthome.core.audio.FileAudioStream;
+import org.eclipse.smarthome.core.voice.TTSException;
+import org.eclipse.smarthome.core.voice.TTSService;
+import org.eclipse.smarthome.core.voice.Voice;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Modified;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.net.URI;
+import java.util.*;
+
+import static org.openhab.voice.gtts.internal.GoogleTTSService.*;
+
+/**
+ * Voice service implementation.
+ *
+ * @author Gabor Bicskei - Initial contribution
+ */
+@Component(configurationPid = SERVICE_PID, property = {
+        Constants.SERVICE_PID + "=" + SERVICE_PID,
+        ConfigurableService.SERVICE_PROPERTY_LABEL + "=" + SERVICE_NAME,
+        ConfigurableService.SERVICE_PROPERTY_DESCRIPTION_URI + "=" + SERVICE_CATEGORY + ":" + SERVICE_ID,
+        ConfigurableService.SERVICE_PROPERTY_CATEGORY + "=" + SERVICE_CATEGORY })
+public class GoogleTTSService implements TTSService, ConfigOptionProvider {
+    /**
+     * Service name
+     */
+    static final String SERVICE_NAME = "Google Cloud TTS Service";
+
+    /**
+     * Service id
+     */
+    static final String SERVICE_ID = "gtts";
+
+    /**
+     * Service category
+     */
+    static final String SERVICE_CATEGORY = "voice";
+
+    /**
+     * Service pid
+     */
+    static final String SERVICE_PID = "org.openhab." + SERVICE_CATEGORY + "." + SERVICE_ID;
```

Comment:
It should actually be directly `org.openhab.<addon-id>`, so please remove the SERVICE_CATEGORY here.